server/src-rsr/migrate_from_13_to_14.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE FUNCTION hdb_catalog.insert_event_log(schema_name text, table_name text, trigger_name text, op text, row_data json) RETURNS text AS $$
+  DECLARE
+  id text;
+  payload json;
+  session_variables json;
+  server_version_num int;
+BEGIN
+  id := gen_random_uuid();
+  server_version_num := current_setting('server_version_num');
+  IF server_version_num >= 90600 THEN
+    session_variables := current_setting('hasura.user', 't');
+  ELSE
+    BEGIN
+      session_variables := current_setting('hasura.user');
+    EXCEPTION WHEN OTHERS THEN
+                session_variables := NULL;
+    END;
+  END IF;
+  payload := json_build_object(
+    'op', op,
+    'data', row_data,
+    'session_variables', session_variables
+  )::text;
```

Comment:
should this be cast as text?

Reply:
You need not. It can be stored as `json` or `jsonb`. Might be useful if you want to query on `event_log` table using some property of `payload` column.

Reply:
The database column is jsonb only. Not sure why I was type casting inside the function though.